src/pages/Editor.tsx

```diff
@@ -1,21 +1,53 @@
 import React from "react";
 import { useForm } from "react-hook-form";
 import Layout from "../components/Layout";
-import { Feed } from "../utils/feed";
+import { useWebnative } from "../context/webnative";
+import * as wn from "webnative";
+import { FilePath } from "webnative/path";
 
 type Inputs = {
   title: string;
   content: string;
+  image: string;
 };
 
-const Editor = () => {
+const Editor = ({ feed }) => {
   const {
     register,
     handleSubmit,
     formState: { errors },
   } = useForm<Inputs>();
 
-  const onSubmit = handleSubmit((data) => console.log(data));
+  const _wn = useWebnative()
+  const { fs } = _wn
+
+  console.log('wn', _wn)
+  console.log('fs', fs)
+  console.log('feed in editor', feed)
+
+  const onSubmit = handleSubmit(async (data) => {
+    console.log('**submit', data)
+
+    if (!fs || !fs.appPath) return
+
+    feed.addItem({
+      // TODO -- how to get id?
+      // could take the hash of the post (without id attribute), then
+      //   add `id: <hash>`
```

Comment:
I've added an issue to address this: https://github.com/fission-suite/blog/issues/7